app.js

```diff
@@ -14,12 +14,44 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
-// Add the route handlers here:
+// Add the route handlers here: 
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then((beersFromApi) => {
+      console.log('Beers from the database: ', beersFromApi);
+      res.render('beers', {beersFromApi});
+    })
+    .catch(error => console.log(error));
+})
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then((responseFromAPI) => {
+      console.log('Random beer from the database: ', responseFromAPI)
+      res.render('random-beer', {responseFromAPI})
+    })
+  .catch(error => console.log(error));
+});
+
+app.get('/beer-detailed', (req, res) => {
+  res.render('beer-detailed')
```

Comment:
There is no need for this route `/beer-detailed` you already have the route `/beers/:id` that is doing the work of rendering the beerDetailed view